qpython/qconnection.py

```diff
@@ -16,26 +16,28 @@
 
 import socket
 import struct
+import ssl
 
 from qpython import MetaData, CONVERSION_OPTIONS
 from qpython.qtype import QException
 from qpython.qreader import QReader, QReaderException
 from qpython.qwriter import QWriter, QWriterException
 
-
+''' SSL Section to load Certificate'''
+context = ssl.SSLContext(ssl.PROTOCOL_TLSv1_2)
```

Comment:
Seems unusual to have a docstring at the top level.
Any reason not to go with `ssl.create_default_context()` ?